aspnet/fundamentals/environments.rst

```diff
@@ -16,7 +16,9 @@ ASP.NET Core introduces improved support for controlling application behavior ac
 Development, Staging, Production
 --------------------------------
 
-ASP.NET Core references a particular `environment variable <https://github.com/aspnet/Home/wiki/Environment-Variables>`_, ``ASPNETCORE_ENVIRONMENT`` (or ``Hosting__Environment`` on \*nix systems), to describe the environment the application is currently running in. This variable can be set to any value you like, but three values are used by convention: ``Development``, ``Staging``, and ``Production``. You will find these values used in the samples and templates provided with ASP.NET Core.
+ASP.NET Core references a particular `environment variable <https://github.com/aspnet/Home/wiki/Environment-Variables>`_, ``ASPNETCORE_ENVIRONMENT`` to describe the environment the application is currently running in. This variable can be set to any value you like, but three values are used by convention: ``Development``, ``Staging``, and ``Production``. You will find these values used in the samples and templates provided with ASP.NET Core.
+
+.. note:: Preview versions of ASP.NET Core used other names for this environment variables so you may encounter those names in documentation. ``ASPNETCORE_ENVIRONMENT`` is the name used in the final release. Older names include ``Hosting:Environment`` and ``ASPNET_ENV``. If you are still using those names, please transition to the final name.
```

Comment:
Yeah, I think this note can be removed. We cover previous incarnations of the environment env var in the [migration](https://docs.asp.net/en/latest/migration/index.html) section. While this note seems relevant now it will become antiquated pretty quickly.